src/preact/base-element.js

```diff
@@ -648,6 +653,7 @@ export class PreactBaseElement extends AMP.BaseElement {
         // to create a simple mechanism that would automatically compute
         // `CanRender = false` on undistributed children.
         addGroup(this.element, UNSLOTTED_GROUP, MATCH_ANY, /* weight */ -1);
+        // eslint-disable-next-line local/restrict-this-access
         setGroupProp(this.element, UNSLOTTED_GROUP, CanRender, this, false);
```

Comment:
Strange, the `this` here doesn't match the type declaration for the 4th param. Should be a `function(T)`. What's going on?

Reply:
I don't follow either. 🙂